addons/l10n_latam_check/views/account_payment_view.xml

```diff
@@ -1,11 +1,34 @@
 <odoo>
 
+    <!-- Own checks search view -->
     <record model="ir.ui.view" id="view_account_payment_search">
         <field name="name">account.check.search</field>
         <field name="model">account.payment</field>
         <field name="priority">20</field>
         <field name="mode">primary</field>
         <field name="inherit_id" ref="account.view_account_payment_search"/>
+        <field name="arch" type="xml">
```

Comment:
@pablohmontenegro @vbe-odoo What is the philosophy for using 2 search views?  Why not 1?

Reply:
@jco-odoo you mean having 1 for third-party checks and another one for own checks?
If that is the case, the purpose of the filter is different:
- Third-party check filter "Checks on hand": when a check is used to pay a vendor bill or is collected, is no longer on hand, and is important to know as it can't be used to pay a vendor bill (so cannot be selected to pay a vendor bill after this happen)
- Own check filter "No Bank Matching": when a check issued by the company is already debited from the bank and that payment is matched with the bank statement line

Or were you referring to something else?
cc' @pablohmontenegro

Reply:
@vbe-odoo @pablohmontenegro It is ok to have two different filters.  The point is why have 2 different search views?  In one tree view you would see the one filter and in the other the other, so why not see them both at once?

Reply:
We keep the 2 views @pablohmontenegro 
As we have 2 diff menus and we need to have diff filters